azure-pipelines.yml

```diff
@@ -112,6 +112,12 @@ jobs:
       continueOnError: true
       condition: always()
 
+- template: eng/common/templates/job/source-build.yml
+  parameters:
+    - platform:
+        name: 'Managed'
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
```

Comment:
This duplicates the contents `eng/common/templates/jobs/source-build.yaml`. Is there some way to just delegate to that (but only for source-build)?